test/config.json

```diff
@@ -1228,6 +1228,24 @@
 			"Shard": "vreplication_migrate",
 			"RetryMax": 1,
 			"Tags": []
+		},
+		"vreplication_source_time_zone": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vreplication", "-run", "TestMoveTablesTZ"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vreplication_migrate",
+			"RetryMax": 1,
+			"Tags": []
+		},
```

Comment:
This change seems unrelated, maybe it should not be in this PR?

Reply:
yeah looks like it added as part of auto conflict resolution .. let me remove it.